### ISSUE TITLE
Quick exit code 127 during release_command

### ISSUE THREAD

**Issue by mrkurt** [superfly/flyctl#708]
I've had several deploys fail this week with logs like this:

```


--> This release will not be available until the release command succeeds.
--
  | Starting instance
  | Unpacking image
  | Starting virtual machine
  | Starting init (commit: 7943db6)...
  | 2022/01/04 19:48:08 listening on [fdaa:0:33:a7b:21e0:5691:f0d5:2]:22 (DNS: [fdaa::3]:53)
  | /usr/src/app/lib/tasks/stripe.rake:87: warning: key :aggregate_usage is duplicated and overwritten on line 93
  | /usr/src/app/lib/tasks/stripe.rake:116: warning: key :aggregate_usage is duplicated and overwritten on line 1
  | 2
  | /usr/src/app/lib/tasks/stripe.rake:148: warning: key :aggregate_usage is duplicated and overwritten on line 155
  | /usr/src/app/config/initializers/acme_alpn.rb:5: warning: already initialized constant Acme::Client::Resources::Challenges::CHALLENGE_TYPES
  | /usr/local/bundle/gems/ruby/2.7.0/gems/acme-client-2.0.6/lib/acme/client/resources/challenges.rb:9: warning: previous definition of CHALLENGE_TYPES was here
  | Main child exited normally with code: 0
  | 🚨 Error: The command exited with status 127
  | user command error: exit status 127
```

This happens fast, like 5 seconds fast. Sometimes they go through but they seem to fail about 2/3 of the time.

**Comment by lubien** [comment]
Might be related, deploying on `ui-ex` today: https://buildkite.com/flyio/ui-ex/builds/257#e2717a0d-98e7-44d2-89f8-05e2f23e9f6b

```sh
==> Monitoring deployment
 
 20 desired, 2 placed, 1 healthy, 0 unhealthy [health checks: 2 total, 2 passing]
 
 20 desired, 20 placed, 20 healthy, 0 unhealthy [health checks: 20 total, 20 passing]
--> v177 deployed successfully
 
--> Troubleshooting guide at https://fly.io/docs/getting-started/troubleshooting/
Error abort
 
🚨 Error: The command exited with status 1
user command error: exit status 1
Running global pre-exit hook
$ /buildkite/hooks/pre-exit
Stopping ssh-agent 14167
Agent pid 14167 killed
```

The changes went online successfully

**Comment by mrkurt** [comment]
I had almost exactly that happen, as well. A deployment that looked like it worked exiting with an error.

**Comment by jsierles** [comment]
Fixed in #710
